docs/src/Callout.doc.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+import * as React from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="Callout"
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'description',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'iconAccessibilityLabel',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'onDismiss',
+        type: '() => void',
+        required: true,
+      },
+      {
+        name: 'primaryLink',
+        type: '{| href: string, label: string |}',
```

Comment:
@norrise120 could we also add an `onClick` handler? That way we can integrate these links with react-router in the Pinterest code base.

Try to use the same type for `onClick` as we use on `<Link />`:

```
  onClick?: AbstractEventHandler<
    | SyntheticMouseEvent<HTMLAnchorElement>
    | SyntheticKeyboardEvent<HTMLAnchorElement>
```